### PR TITLE
Feature/email campaigns

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**

### CODE REVIEW

File: practice-two/src/assets/styles/layouts/_card.scss [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      .card-title {
+        font-size: 28px;
+        line-height: 31px;
+        margin-bottom: 17px;
+      }
+    }
+  }
+
+  .campaigns-card:nth-child(2) {
+    float: right;
```
Comment (phuocduong-agilityio) [inline practice-two/src/assets/styles/layouts/_card.scss:52]: Avoid to use float

File: practice-two/src/assets/styles/layouts/_card.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  }
+}
+
+@include tablet {
+  .email-campaigns {
```
Comment (phuocduong-agilityio) [inline practice-two/src/assets/styles/layouts/_card.scss:57]: Typo?

File: practice-two/src/assets/styles/layouts/_card.scss [hunk 3]
```diff
@@ -0,0 +1,117 @@
+.email-campaigns {
```
Comment (phuocduong-agilityio) [inline practice-two/src/assets/styles/layouts/_card.scss:1]: This is not a place for email campaigns, pls correct it

File: practice-two/src/assets/styles/layouts/_campaigns.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        .btn-link {
+          font-size: $font-size-x-small;
+          line-height: 19px;
+        }
+
+        .card-title {
+          font-size: $font-size-x-large;
+          line-height: $line-height-x-large;
+        }
+
```
Comment (phuocduong-agilityio) [inline practice-two/src/assets/styles/layouts/_campaigns.scss:98]: Remove empty line, only add empty line between 2 groups/blocks styles

File: practice-two/src/assets/styles/layouts/_campaigns.scss [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+      }
+    }
+    .campaigns-card:nth-child(2) {
```
Comment (phuocduong-agilityio) [inline practice-two/src/assets/styles/layouts/_campaigns.scss:101]: Add empty line

File: practice-two/src/assets/styles/layouts/_campaigns.scss [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    .campaigns-card {
+      .card-img {
+        width: 320px;
+        height: 282px;
+      }
+
+      .card-body {
+        width: 278px;
+
+        &-bottom {
```
Comment (phuocduong-agilityio) [inline practice-two/src/assets/styles/layouts/_campaigns.scss:79]: bottom is not suitable name. Please use another name

File: practice-two/src/assets/styles/layouts/_campaigns.scss (hunk 5 above)
Comment (phuocduong-agilityio) [reply practice-two/src/assets/styles/layouts/_campaigns.scss:101]: Btw, is there anyway to set width without use child?

Comment (daoanhhuy26012001) [reply practice-two/src/assets/styles/layouts/_campaigns.scss:101]: add a new class, but I think it will be better to do it like this

**Pull request merged**